go.mod

```diff
@@ -604,7 +604,7 @@ require github.com/lorenzosaino/go-sysctl v0.3.1
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.2
-	github.com/DataDog/agent-payload/v5 v5.0.115
+	github.com/DataDog/agent-payload/v5 v5.0.116-0.20240515080026-e5bfdb65d50b
```

Comment:
You should create a release of the payload https://github.com/DataDog/agent-payload/?tab=readme-ov-file#publishing-changes